src/lib/journey/stages/generic.svelte

```diff
@@ -1,34 +1,37 @@
 <script lang="ts">
   import type { FRLoginFailure, FRLoginSuccess, FRStep } from '@forgerock/javascript-sdk';
-  import type { Writable } from 'svelte/store';
 
   // Import primitives
   import Button from '$components/primitives/button/button.svelte';
   import Form from '$components/primitives/form/form.svelte';
-  import KeyIcon from '$components/icons/key-icon.svelte';
+  import T from '$components/i18n/index.svelte';
   import { mapCallbackToComponent } from '$journey/utilities/map-callback.utilities';
+  import Spinner from '$components/primitives/spinner/spinner.svelte';
 
   type StepTypes = FRStep | FRLoginSuccess | FRLoginFailure | null;
 
   export let formEl: HTMLFormElement | null = null;
-  export let step: Writable<StepTypes>;
+  export let step: StepTypes;
   export let submitForm: () => void;
 </script>
 
-<div class="tw_flex tw_justify-center">
-  <KeyIcon classes="tw_text-gray-400 tw_fill-current" size="72px" />
-</div>
-<h1 class="tw_primary-header dark:tw_primary-header_dark">Sign In</h1>
-
-{#if !$step}
-  <p>Loading ...</p>
-{:else if $step.type === 'Step'}
+{#if !step}
+  <div class="tw_text-center tw_w-full tw_py-4">
+    <Spinner colorClass="tw_text-primary-light" layoutClasses="tw_h-28 tw_w-28" />
+  </div>
+{:else if step.type === 'Step'}
+  <h1 class="tw_primary-header dark:tw_primary-header_dark">
+    <!-- TODO: Needs localization strategy -->
+    {step.getHeader() || ''}
```

Comment:
Are we allowing HTML to be rendered from within the header as defined in AM? I have seen this before, I think even on default journeys. Not sure if it's a convention or expectation that a UI will render this as platform-login does. Perhaps having this as an option for the app developer to turn on / off could be good, when we do implement localization for this bit.

Reply:
The challenge here is that many of these strings that can be set in AM, like this header value, don't provide alternative language values. But yes, the values of `header` and `description` can be regular strings or HTML, but in situations where localization matters, these values are hardcoded to a single language, so they can't be used.

My suggestion would be to just use the `header` and `description` value coming from AM (page node to be exact) as the key itself. So, rather than the `header` value being `"Sign Up, Today!"`, for example, it would be `"signUpToday"`.

> Perhaps having this as an option for the app developer to turn on / off could be good

We might be able to do this, but I think it's a different user story. The downside to this is we lose the single source of truth for content. Some things have to be provided by the content files; some things that come from AM don't have prompts or labels, some things do but only in a single language, some things have multiple languages ... it's just keeping track of all of that in a resilient way.

I think we might be able to come to a better conclusion once I can speak with Mike.